packages/qemu-system-x86-64-headless/build.sh

```diff
@@ -33,6 +35,8 @@ termux_step_configure() {
 	QEMU_TARGETS+="aarch64-linux-user,"
 	QEMU_TARGETS+="arm-linux-user,"
 	QEMU_TARGETS+="i386-linux-user,"
+	QEMU_TARGETS+="ppc64-linux-user"
+	QEMU_TARGETS+="ppc-linux-user"
```

Comment:
You have fixed the missing comma for system targets, but what's for user?

Reply:
Fixed